File: Main-service/src/main/java/com/legion/externalMicroservices/crm/identityObjects/BandData.java
package com.legion.externalMicroservices.crm.identityObjects;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.time.Instant;
import java.util.UUID;

@NoArgsConstructor
@Getter
@Setter
public class BandData {
    private UUID id;
    private String name;
    private String city;
    private String phoneNumber;
    private String genre;
    private String description;
    private Instant createDate;
    private Instant updateDate;
    private boolean removed;
}

File: crm/src/main/kotlin/com/legion/band/entity/BandData.kt
package com.legion.band.entity

import com.legion.common.CrmEntity
import com.legion.common.DataEntity
import com.legion.user.entity.User
import java.time.Instant
import java.util.*
import javax.persistence.*

@Entity
@Table(name = "band_data")
data class BandData (

        @Id
        @Column(name = "id", unique = true)
        override val id: UUID = UUID.randomUUID(),

        @Column(name = "name")
        override val name: String,

        @Column(name = "city")
        override val city: String,

        @Column(name = "phone_number")
        val phoneNumber: String?,

        @Column(name = "genre")
        val genre: String,

        @Column(name = "description")
        val description: String,

        @OneToOne(fetch = FetchType.LAZY)
        @JoinColumn(name = "user_id")
        val user: User,

        @Column(name = "creation_date")
        override val createDate: Instant = Instant.now(),

        @Column(name = "update_date")
        override val updateDate: Instant? = Instant.now(),

        @Column(name = "removed")
        override val removed: Boolean = false

) : CrmEntity, DataEntity